controllers.js

```diff
@@ -52,8 +52,21 @@ exports.postCommentByArticleId = (req, res, next) => {
   models
     .insertCommentByArticleId(req.body, req.params.article_id)
     .then((comment) => {
-      console.log(comment);
       res.status(201).send({ comment });
     })
     .catch(next);
 };
+
+exports.patchArticleById = (req, res, next) => {
+  models
+    .updateArticleById(req.body.inc_votes, req.params.article_id)
+    .then((article) => {
+      if (article) {
+        res.status(201).send(article);
+      } else {
+        // if index[0] undefined
```

Comment:
It would be better to define what situation you want this error to be triggered, rather than putting it in an else block as this leaves it open to the 404 being triggered when it wasnt meant to be